packages/release/media-cdrom.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CDROM mount(/media/cdrom)
+Requires=dev-cdrom.device
+After=dev-cdrom.device
+
+[Mount]
+What=/dev/cdrom
+Where=/media/cdrom
+Type=iso9660
+Options=defaults,noexec,context=system_u:object_r:private_t:s0
```

Comment:
@bcressey : does the `context` look correct here?

Reply:
What label does `ls -lZ` show if you don't specify this option?

I'd say `external_t` is more correct and possibly that's the default.

Reply:
Without the option the label is `unlabeled_t`

Reply:
Per offline conversation, `unlabeled_t` should be fine. I removed the `context=` from the line.